<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mixer/Composer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #000;
padding: 6px 12px;
border: 1px solid rgba(0, 0, 0, 0.20);
border-radius: 5px;

/* Body 2 */

font-family: 'Roboto';
font-style: normal;
font-weight: 400;
font-size: 12px;
line-height: 20px;
/* identical to box height, or 143% */

letter-spacing: 0.25px;

/* 00. On Surface / High Emphasis */

/*color: rgba(0, 0, 0, 0.50);*/
color: black;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="PlotWidget" name="sin_graphics_view"/>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Frequency</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="signalPhase"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="signalMagnitude"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Magnitude</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Phase</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="signalFrequency"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="signalName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="plot_push_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton {background-color:white;}
::hover {
    background-color: rgb(238, 238, 238); 
     border: 1px solid rgb(10, 10, 10);
    
    color: black; 
}


</string>
           </property>
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_push_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton {background-color:white;}
::hover {
    background-color: rgb(238, 238, 238); 
     border: 1px solid rgb(10, 10, 10);
    
    color: black; 
}</string>
           </property>
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
